### DISCUSSION TITLE
Documentation inconsistency with static assets

### DISCUSSION THREAD

**Discussion by pat-son** [facebook/docusaurus#7845] [Q&A] [answered]
One page recommends static assets (e.g. images) go in the /static folder: https://docusaurus.io/docs/static-assets
Another says that it's convenient to colocate your images with the docs page: https://docusaurus.io/docs/markdown-features/assets

So which is better? Is there a heuristic to decide when to do one over the other?

**Comment by Josh-Cena** [comment, marked as answer]
You are encouraged to colocate them where you can—especially images and file assets. That static asset page was created before asset colocation was made as ergonomic as today. Feel free to change "By default, you are suggested to put these assets in the static folder." to something like "The static folder is designed to hold assets".

(But note there are some nuances between static and non-static assets)

**Reply by pat-son** [reply]
Thanks.

> But note there are some nuances between static and non-static assets

Any important nuances besides the way you import them?
